### DISCUSSION TITLE
Import AGeNT in sarek

### DISCUSSION THREAD

**Discussion by RP-Bioinfo** [nextflow-io/nextflow#5259] [General]
Hi guys!

I'm a researcher dealing with the secondary analysis of targeted and whole-exome data. Since we use the Agilent SureSelect XT HS2 kits, which adds to sequencing libraries​ the duplex molecular barcodes, we need AGeNT tool (Agilent) for trimming and deduplication of reads. Is there a way to incorporate this tool into sarek? Thanks in advance for your reply.

Best wishes.

Riccardo

**Comment by bentsherman** [comment]
Please submit an issue on the sarek repo: https://github.com/nf-core/sarek/issues
